examples/custom-server-typescript/package.json

```diff
@@ -7,15 +7,14 @@
     "start": "cross-env NODE_ENV=production node dist/index.js"
   },
   "dependencies": {
-    "next": "^8.1.0",
+    "next": "^8.1.1-canary.69",
     "react": "^16.8.4",
     "react-dom": "^16.8.4"
   },
   "devDependencies": {
     "@types/next": "^8.0.5",
```

Comment:
We don't need this anymore since Next.js typings are included by default. We can replace this with `@types/node`.